state/elasticache-redis.yaml

```diff
@@ -124,6 +125,11 @@ Parameters:
     Default: 1
     MinValue: 0
     MaxValue: 5
+  MultiAZ:
```

Comment:
I would argue, that MultiAZ should be enabled by default when NumReplicas > 0? Can you think of any reason to disable MultiAZ?

Reply:
The docs mention this:

> Multi-AZ is only supported on Redis clusters that have more than one node in each shard.